src/foam/nanos/approval/Approvable.js

```diff
@@ -90,6 +90,13 @@ foam.CLASS({
       class: 'Boolean',
```

Comment:
@tharmaman can you document this one, please.

Reply:
donee.